foursight_core/app_utils.py

```diff
@@ -163,14 +258,132 @@ def check_authorization(self, request_dict, env=None):
                 for env_info in self.init_environments(env).values():
                     user_res = ff_utils.get_metadata('users/' + payload.get('email').lower(),
                                                      ff_env=env_info['ff_env'], add_on='frame=object')
-                    logger.error(env_info)
-                    logger.error(user_res)
-                    if not ('admin' in user_res['groups'] and payload.get('email_verified')):
+                    logger.warn("foursight_core.check_authorization: env_info ...")
+                    logger.warn(env_info)
+                    logger.warn("foursight_core.check_authorization: user_res ...")
+                    logger.warn(user_res)
+                    logger.warn("foursight_core.check_authorization: payload ...")
+                    logger.warn(payload)
```

Comment:
Warnings are not like print statements. There is not always a 'next line'. If you pump warnings into ES and search for them using Kibana or something like that, you will perhaps find one warning but not the warning right after it, making things that depend on doing multi-line output like this make no sense, which is why you see me consolidating warnings onto a single line:
```
logger.warn(f"foo:")
logger.warn(f"   {detailed_foo}")
```
becoming
```
logger.warn(f"foo: {detailed_foo}")
```
even where the line is long.

Reply:
Okay, yeah, I understand, thanks. Removing some of those too as I'd added for troubleshooting a problem.